README.md

```diff
@@ -99,13 +111,18 @@ Note that in the following instructions, we assume your catkin workspace (in whi
    mkdir -p ~/catkin_ws/src
    ```
 
-1. Install mavros version 0.29.0 or above. Instructions to install it from sources can be found here: https://dev.px4.io/en/ros/mavros_installation.html. If you want to install using apt, be sure to check that the version is 0.29.0 or greater.
-
+1. Install mavros version 0.29.0 or above. Instructions to install it from sources can be found here: https://dev.px4.io/en/ros/mavros_installation.html.
```

Comment:
How do you tell what version you have in apt - ie to check you have more than 0.29.0?? 
I tried several versions but the version string I get is `ros-melodic-mavros:amd64/bionic 1.0.0-1bionic.20200121.134842 uptodate`